lib/notification-issue.coffee

```diff
@@ -174,12 +174,13 @@ class NotificationIssue
     return packageName if packageName
 
   getPackageName: ->
+    message = @notification.getMessage()
     options = @notification.getOptions()
-    return unless options.stack? or options.detail?
+    return unless message? or options.stack? or options.detail?
 
     packagePaths = @getPackagePathsByPackageName()
     for packageName, packagePath of packagePaths
-      if packagePath.indexOf('.atom/dev/packages') > -1 or packagePath.indexOf('.atom/packages') > -1
+      if packagePath.indexOf(path.join('.atom', 'dev', 'packages')) > -1 or packagePath.indexOf(path.join('.atom', 'packages')) > -1
```

Comment:
I think it's actually to check if a package is symlinked or not, and then to get its true path since that's where the error will have been thrown from.

Eg if I have notifications linked to `.atom\packages\notifications` from `C:\Users\user\Documents\Github\notifications` and notifications throws an error, then this piece of code will correctly match the two.